src_testbed/testbed.rs

```diff
@@ -886,16 +899,16 @@ impl Testbed<'_, '_, '_, '_, '_, '_> {
 
             let controller = self.state.character_controller.unwrap_or_default();
             let phx = &mut self.harness.physics;
-            let character_body = &phx.bodies[character_handle];
-            let character_collider = &phx.colliders[character_body.colliders()[0]];
+            let character_body = &phx.context.bodies[character_handle];
+            let character_collider = &phx.context.colliders[character_body.colliders()[0]];
```

Comment:
to avoid adding those, maybe events and hooks should be directly in the harness ? or in a different property ?